### DISCUSSION TITLE
How to successfully connect Huawei 4G USB modems in Static IP ?

### DISCUSSION THREAD

**Discussion by gaamaaresosa** [Ysurac/openmptcprouter#2757] [Q&A]
I need to bond 2 numbers of Huawei (HUAWEI E8372 (4G/LTE) WiFi Wingle is a 4G dongle) 4G 
USB modem with OpenMPTCProuter on Raspberry Pi 4.

I use an external powered 8 port USB hub to connect all 2 Huawei modems on RPi 4 USB port.

When I enable DHCP on all 2 modems and DHCP as well on OMR interface, 
I could see all the 2 modems sucessfully activated on the OMR status window.

But you recommend for Static IP on all 2 modems and OMR interfaces.
So I logged in on each Huawei modem from my PC and disabled DHCP and set 
Huawei_1 IP = 192.168.11.1
Huawei_2 IP = 192.168.10.1

Then connected these 2 modems on the usb hub
The usb hub to RPi 4.
RPi 4 LAN is connected to my Windows PC for configuring OpenMPTCProuter.

What ever I try I don't see the modems successfully connected on Static IP.

Your precious help is highly appreciated and thanks in advance.

Model used:
HUAWEI E8372 (4G/LTE) WiFi Wingle is a 4G dongle 
Raspberry Pi 4B 

NOTE: Images attached:

![LAN](https://user-images.githubusercontent.com/21296884/216431835-a1665b10-7d49-4d5e-a714-0df2f5316f0b.png)
![WAN1](https://user-images.githubusercontent.com/21296884/216431838-330530d1-ff8d-49d3-a2eb-ced5e4a2b301.png)
![WAN2](https://user-images.githubusercontent.com/21296884/216431841-f0e59ddc-0789-44c2-a2af-0c7f441efd37.png)

![Huawei Static IP](https://user-images.githubusercontent.com/21296884/216429101-4ff88cbf-c315-4410-aa92-6f19e8746be7.png)
<img width="521" alt="HUAWEI E8372_2" src="https://user-images.githubusercontent.com/21296884/216430770-c93183dc-4dbb-4d8b-b6b0-1eb438ef3d56.png">
![HUAWEI E8372](https://user-images.githubusercontent.com/21296884/216430778-6d50ff14-095e-4cb5-aa98-516f671666b2.jpg)

**Comment by Ysurac** [comment]
For this modem, you can keep it using DHCP, there is a local ip and wan IP, it doesn't directly give WAN ip to interface, so no real renew.
I own same usb key.

**Reply by gaamaaresosa** [reply]
Thanks you so much for your super swift responce Ysurac.
Ok, I will enable DHCP for the Huawei modems.
What about OMR interface settings, that too DHCP for these modems or static as I set ?

**Comment by gaamaaresosa** [comment]
Btw how to test the upload speed properly ?
Is there any speed test on UI ?

I tried below commands and not sure where to see the upload speed.
omr-test-speed
omr-test-speed eth1
omr-test-speed eth2
omr-iperf

![SpeedTest omr-iperf](https://user-images.githubusercontent.com/21296884/216457112-b6a746e7-116e-4dec-b005-c8f79a88fa12.png)


![SpeedTest Wan1 and Wan2](https://user-images.githubusercontent.com/21296884/216456552-099e2e31-6332-4ad0-b05c-353856cf4dcf.png)

![SpeedTest](https://user-images.githubusercontent.com/21296884/216456598-04ed4d37-e083-4fe9-9e9a-cc3a6098af18.png)
